crates/chia-consensus/src/gen/solution_generator.rs

```diff
@@ -39,14 +42,92 @@ where
     Ok(quote)
 }
 
+// the tuple has the Coin, puzzle-reveal and solution
+fn build_generator_no_backrefs<BufRef, I>(a: &mut Allocator, spends: I) -> io::Result<NodePtr>
```

Comment:
the existing `build_generator()` function (above) is private in this file, and only used in one place. I don't think there's any need to preserve it. i.e. I think it would be simpler to just change it to not parse `solution` and `puzzle` supporting backrefs